tests/integration/test_source_git.py

```diff
@@ -83,6 +83,10 @@ def test_basic_local_update_patch_content(
     source_file.write_text("new changes")
     git_add_and_commit(directory=sourcegit, message="source change")
 
+    source_file = sourcegit / "ignored_file.txt"
+    source_file.write_text(" And I am sad.")
+    git_add_and_commit(directory=sourcegit, message="make a file sad")
```

Comment:
Can there be some explicit assert that will check the absence in the patch?

Reply:
There are two asserts below which check that there is only a single patch included in the spec - I guess that should take care of this as well. So should I also check the patch itself?

Reply:
oh, now I get it; this is what I have:
```
    assert "--- a/ignored_file.txt\n" not in git_diff
```
but forgot to push it here, it's in #658